README.md

```diff
@@ -131,7 +131,15 @@ highlighting:
 git show v0.6.0:src/main.rs | bat -l rs
 ```
 
-Note that syntax highlighting within diffs is currently not supported. If you are looking for this, check out [`delta`](https://github.com/dandavison/delta).
+#### `git diff`
+
+You can combine `bat` with `git diff` to view lines around code changes with proper syntax
+highlighting:
+```bash
+git diff --name-only --diff-filter=d | xargs bat --diff
+```
```

Comment:
@eth-p Probably also a good place to refer to `batdiff` from `bat-extras`, right?

Reply:
But we can already merge this as-is.

Reply:
This would be a good place to refer to it, yeah.